bin/run.js

```diff
@@ -5,15 +5,19 @@ const resolve = require('path').resolve
 
 program
   .version("0.0.1")
-  .option("-c, --config [dextrose-config]", "please specify the dextrose config")
+  .option("-c, --config [dextrose-config]", "A file that exports the dextrose config for your app")
+  .option("-t, --snapshotWait [snapshot-wait]", "the amount of timeout to wait between loading a component and taking the snap", parseInt)
+  .option("-T, --timeout [timeout]", "the timeout applied to appium before it closes the app", parseInt)
   .parse(process.argv);
 
 
-if (!program.config)
-  throw new Error(
-    "Please specify the dextrose config: --config [directory/config.js]"
-  );
+if (!program.config){
+  console.log("Please specify the dextrose config: --config [directory/config.js]");
```

Comment:
Does this need to be a warn or error or something?

Reply:
it's a message about why we can't execute the command. Maybe we should use a proper logger

Reply:
Ah yeah I see, ok well it's fine for now but we should use a proper logger. I will raise a separate issue.

Reply:
we have got a proper logger in dextrose. It's called `logger` under `src`